frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -592,23 +590,46 @@ private void addReleaseBreadcrumb(RenderRequest request, RenderResponse response
 
     private void prepareStandardView(RenderRequest request) throws IOException {
         String searchtext = request.getParameter(KEY_SEARCH_TEXT);
-
         String searchfilter = request.getParameter(KEY_SEARCH_FILTER_TEXT);
 
+        List<Component> componentList = getFilteredComponentList(request);
+
+        Set<String> vendorNames;
+
+        try {
+            vendorNames = thriftClients.makeVendorClient().getAllVendorNames();
+        } catch (TException e) {
+            log.error("Problem retrieving all the Vendor names");
```

Comment:
also log the exception? i.e. `log.error("Problem retrieving all the Vendor names",e);`